submitted_models/emesent_hovermap_sensor_config_1/model.sdf

```diff
@@ -0,0 +1,774 @@
+          <ixx>0.00058083</ixx>
+          <ixy>0</ixy>
+          <ixz>0</ixz>
+          <iyy>0.0005</iyy>
+          <iyz>0</iyz>
+          <izz>0.00059083</izz>
+        </inertia>
+      </inertial>
+      <collision name='gimbal_roll_collision'>
+        <pose>0 0 0 0 -0 0</pose>
+        <geometry>
+          <box>
+            <size>0.001 0.001 0.001</size>
+          </box>
+        </geometry>
+        <surface>
+          <contact>
+            <ode/>
+          </contact>
+          <friction>
+            <ode/>
+          </friction>
+        </surface>
+      </collision>
+      <visual name='gimbal_roll_visual'>
+        <pose>0 0 0 0 -0 0</pose>
+        <geometry>
+          <mesh>
+            <scale>1 1 1</scale>
+            <uri>meshes/gimbal_roll.dae</uri>
+          </mesh>
+        </geometry>
+      </visual>
+      <gravity>1</gravity>
+      <velocity_decay/>
+    </link>
+    <joint name='gimbal_roll_joint' type='revolute'>
+      <child>gimbal_roll</child>
+      <parent>gimbal_pan</parent>
+      <axis>
+        <xyz>1 0 0</xyz>
+        <limit>
+          <lower>-0.785398</lower>
+          <upper>0.785398</upper>
+          <effort>10</effort>
+          <velocity>20</velocity>
+        </limit>
+        <dynamics>
+          <spring_reference>0</spring_reference>
+          <spring_stiffness>0</spring_stiffness>
+        </dynamics>
+        <use_parent_model_frame>0</use_parent_model_frame>
+      </axis>
+    </joint>
+    <link name='gimbal_tilt'>
+      <pose>0.218 0 0.125 0 -0 0</pose>
+      <inertial>
+        <pose>-2.4e-05 0.002445 -2e-05 0 -0 0</pose>
+        <mass>0.0001</mass>
+        <inertia>
+          <ixx>0.00216206</ixx>
+          <ixy>0</ixy>
+          <ixz>0</ixz>
+          <iyy>0.00106169</iyy>
+          <iyz>0</iyz>
+          <izz>0.00218206</izz>
+        </inertia>
+      </inertial>
+      <collision name='gimbal_tilt_collision'>
+        <pose>0 0 0 0 -0 0</pose>
+        <geometry>
+          <box>
+            <size>0.001 0.001 0.001</size>
+          </box>
+        </geometry>
+        <surface>
+          <contact>
+            <ode/>
+          </contact>
+          <friction>
+            <ode/>
+          </friction>
+        </surface>
+      </collision>
+      <visual name='gimbal_tilt_visual'>
+        <pose>0 0 0 0 -0 0</pose>
+        <geometry>
+          <mesh>
+            <scale>1 1 1</scale>
+            <uri>meshes/gimbal_tilt.dae</uri>
+          </mesh>
+        </geometry>
+      </visual>
+      <visual name='gimbal_tilt_fixed_joint_lump__gimbal_flashlight_visual_2'>
+        <pose>-0.012 -0.025 -0.01 0 0 -1.57</pose>
+        <geometry>
+          <mesh>
+            <scale>0.005 0.005 0.005</scale>
+            <uri>meshes/spotlight.dae</uri>
+          </mesh>
+        </geometry>
+      </visual>
+      <velocity_decay/>
+      <velocity_decay/>
+      <sensor name='gimbal_camera' type='camera'>
+        <update_rate>10</update_rate>
```

Comment:
We typically set the frame rate to about 5-10 Hz. I believe this is configurable through the camera driver and limits the camera hardware to that rate. This is because our detectors on hovermap cannot process high framerates and are sort of limited to about 3Hz, depending on the detector.